pkg/ipamd/ipamd.go

```diff
@@ -1771,10 +1767,18 @@ func getMinimumIPTarget() int {
 	return noMinimumIPTarget
 }
 
-func disablingENIProvisioning() bool {
+func disableENIProvisioning() bool {
 	return getEnvBoolWithDefault(envDisableENIProvisioning, false)
 }
 
+func disableLeakedENICleanup() bool {
```

Comment:
Can we rename this? I am not getting a good variable name but something like `disableENIOperations`

Reply:
This is used exclusively for disabling the leaked ENI cleanup task, though. It doesn't disable other ENI operations. `disableENIProvisioning` is the flag we use for cases like not allocating or deallocating an ENI

Reply:
Will sync up offline on this..